### DISCUSSION TITLE
How to find out which site had changes in globals?

### DISCUSSION THREAD

**Discussion by lotarbo** [statamic/cms#8274] [Q&A]
Hello, i use statamic multisite as headless cms for many sites.
I run pipeline with nextjs export command for any updates.
I can detect site when entry updated in events and run pipeline only for this site.
But when global set updates, it has localizations prop with all sites.
I use GlobalSetSaved for this. Mb there are some trick to do it?

**Comment by jasonvarga** [comment]
You can't at the moment. The global set object as a whole is what gets saved.
